controlsfx/build.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'biz.aQute.bnd.builder'
```

Comment:
This is https://github.com/bndtools/bnd right?

Apache License 2 so should be good for us.

Reply:
Yeah!